### DISCUSSION TITLE
Why is the engine stopped on the opponent’s turn?

### DISCUSSION THREAD

**Discussion by ghost** [official-stockfish/Stockfish#5732] [Q&A] [answered]
When you look at the TCEC, you can see that when stockfish has played, he stops calculation. But can’t we use the computational time when it’s the opponent’s turn?

**Comment by Disservin** [comment, marked as answer]
This feature is called pondering, as is typically disabled at every tournament, both engines require the full power of the machine but this doesn't work when both engines are running.

**Reply by ghost** [reply]
> This feature is called pondering, as is typically disabled at every tournament, both engines require the full power of the machine but this doesn't work when both engines are running.

If I understood, it’s implemented but restricted by many tournaments?

**Reply by cj5716** [reply]
Yes, it is an unused feature most of the time
